inc/views/base/postbit/postbit.twig

```diff
@@ -0,0 +1,218 @@
+	<script type="text/javascript">
+	<!--
+		$('#show_ignored_link_{{ post.pid }}').show();
+	// -->
+	</script>
+			{{ post.ignored_message|raw }}
+		</div>
+	</div>
+{% endif %}
+{% if post.isdeleted %}
+	<div class="deleted_post_collapsed" id="deleted_post_{{ post.pid }}">
+		<div class="deleted_post_author"><strong><span class="largetext">{{ post.profilelink|raw }}</span></strong></div>
+		<div class="deleted_post_message">
+			<div class="show_deleted_post float_right" style="vertical-align: top;" id="show_deleted_link_{{ post.pid }}"><a href="javascript:void(0)" onclick="Thread.showDeletedPost({{ post.pid }}); return false;" class="button small_button"><span>{{ lang.postbit_show_ignored_post }}</span></a></div>
+	<script type="text/javascript">
+	<!--
+		$('#show_deleted_link_{{ post.pid }}').show();
+	// -->
+	</script>
+			{{ post.deleted_message }}
+		</div>
+	</div>
+{% endif %}
+<a name="pid{{ post.pid }}" id="pid{{ post.pid }}"></a>
+<div class="post classic {{ post.unapproved_shade }}" style="{{ post.visibility }}" id="post_{{ post.pid }}">
+	<div class="post_author scaleimages">
+		{% if post.showavatar %}
+			<div class="author_avatar">
+				<a href="{{ post.profilelink_plain|raw }}"><img src="{{ post.useravatar.image }}" alt="" {{ post.useravatar.width_height|raw }} /></a>
+			</div>
+		{% endif %}
+		<div class="author_information">
+			<strong><span class="largetext">{{ post.profilelink|raw }}</span></strong> 
+			{% if post.onlinestatus == "online" %}
+				<a href="online.php" title="{{ lang.postbit_status_online }}"><img src="{{ theme.imgdir }}/buddy_online.png" border="0" alt="{{ lang.postbit_status_online }}" class="buddy_status" /></a>
+			{% elseif post.onlinestatus == "away" %}
+				<a href="{{ post.profilelink_plain|raw }}" title="{{ lang.postbit_status_away }}"><img src="{{ theme.imgdir }}/buddy_away.png" border="0" alt="{{ lang.postbit_status_away }}" class="buddy_status" /></a>
+			{% else %}
+				<img src="{{ theme.imgdir }}/buddy_offline.png" title="{{ lang.postbit_status_offline }}" alt="{{ lang.postbit_status_offline }}" class="buddy_status" />
+			{% endif %}
+<br />
+				<span class="smalltext">
+					{{ post.usertitle|raw }}<br />
+					{% if post.starimage and post.stars %}
+						{% for i in range(0, post.stars) %}
+							<img src="{{ post.starimage }}" border="0" alt="*" />
+						{% endfor %}
+						<br />
+					{% endif %}
+					{% if post.groupimage %}
+						<img src="{{ usergroup.image }}" alt="{{ usergroup.title }}" title="{{ usergroup.title }}" /><br />
+					{% endif %}
+				</span>
+		</div>
+		<div class="author_statistics">
+			{% if post.isguest %}
+				&nbsp;
+			{% else %}
+				{{ lang.postbit_posts }} {{ post.postnum }}<br />
+				{{ lang.postbit_threads }} {{ post.threadnum }}<br />
+				{{ lang.postbit_joined }} {{ post.userregdate }}
+				{% if post.replink %}
+					<br />{{ lang.postbit_reputation }} {{ post.userreputation|raw }}
+				{% endif %}
+				{% for field in post.profile_fields %}
```

Comment:
Does this result in all of the profile fields being shown in the postbit? I believe these fields are currently 'optional' -- you can add a variable to include them if you want, but they aren't included by default.

(That said, the 1.9 theme concepts don't have _any_ fields in the postbit...)

Reply:
@justinsoltesz the profile fields that are to be displayed are loaded before `build_postbit` is called. In other words, it is a global variable `$profile_fields` that the calling code would define.

Reply:
Sorry, you're right. I didn't realize they were automatically added based on a setting in the ACP in 1.8 (they definitely weren't in a previous version).

Reply:
No problem. I am getting to know this section of code much better than I ever have before. I guess that is another benefit of this conversion as well as some long overdue code cleanup and convention updates.